pkg/exchange/binance/exchange.go

```diff
@@ -45,17 +46,25 @@ type Exchange struct {
 	types.FuturesSettings
 
 	key, secret string
-	Client      *binance.Client
+	Client			*binance.Client // Spot & Margin 
+	futuresClient	*futures.Client // USDT-M Futures
+	// deliveryClient	*delivery.Client // Coin-M Futures
 }
 
 func New(key, secret string) *Exchange {
-	var client = binance.NewClient(key, secret)
-	_, _ = client.NewSetServerTimeService().Do(context.Background())
+	var Client = binance.NewClient(key, secret)
+	var futuresClient = binance.NewFuturesClient(key, secret)
+	Client.HTTPClient = &http.Client{Timeout: 15 * time.Second}
+	futuresClient.HTTPClient = &http.Client{Timeout: 15 * time.Second}
+	_, _ = Client.NewSetServerTimeService().Do(context.Background())
+	_, _ = futuresClient.NewSetServerTimeService().Do(context.Background())
```

Comment:
should check error here? if err then panic

Reply:
Done. Thx!